example/presto_standalone/README.md

```diff
@@ -0,0 +1,31 @@
+# Presto cluster example
+The current directory contains terraform related files that use the module in `../../`. The example module spins up presto in [standalone mode](../conf/nomad/presto_standalone.hcl
+) having one node work as both coordinator and worker.
+It uses vault as the shared secret provider. For more details check [main.tf](./main.tf).
```

Comment:
Regarding the presto_standalone folder (not this readme), you need to update the table in the [example readme](https://github.com/fredrikhgrelland/terraform-nomad-presto/tree/standalone/example) with this new example.

Reply:
Ah thanks. Done!